redis/redis.conf

```diff
@@ -0,0 +1,2326 @@
+# Redis configuration file example.
```

Comment:
Do we need this whole config file, or can we just overwrite the default configuration somehow?

Reply:
Seems to work as just config file with the single no save prop override. I've updated